_posts/2023-01-11-#26742.md

```diff
@@ -7,18 +7,59 @@ authors: [fjahr]
 components: ["rpc/rest/zmq"]
 host: stickies-v
 status: upcoming
-commit:
```

Comment:
Re: title, I wonder if it's worth stripping "2nd attempt" and/or "http" ? I usually remove stuff if already labeled in components